src/Kafka/CreateKafkaInstanceWithSizes/Stories/storiesHelpers.tsx

```diff
@@ -50,76 +49,113 @@ export const AZURE: ProviderInfo = {
 
 export const PROVIDERS: Providers = [AWS, AZURE];
 
-const SIZES: { [key: string]: Size[] } = {
-  aws: [
-    {
-      id: "x1",
+const SIZES: { [key: string]: GetSizesData } = {
+  aws: {
+    standard: [
+      {
+        id: "x1",
+        displayName: "1",
+        quota: 1,
+        status: "stable",
+        ingress: 3,
+        egress: 31,
+        storage: 5,
+        connections: 6,
+        connectionRate: 7,
+        maxPartitions: 8,
+        messageSize: 9,
+        trialDurationHours: undefined,
+      },
+      {
+        id: "x2",
+        displayName: "2",
+        quota: 2,
+        status: "preview",
+        ingress: 30,
+        egress: 301,
+        storage: 50,
+        connections: 60,
+        connectionRate: 70,
+        maxPartitions: 80,
+        messageSize: 90,
+        trialDurationHours: undefined,
+      },
+      {
+        id: "x3",
+        displayName: "3",
+        quota: 5,
```

Comment:
I'm not sure I fully understand how this data relates to the implementation in production, so this comment might be irrelevant.

One of the updates we discussed for the release we want to make this week is to use `displayName` for the option label on the slider, but still use `quotaConsumed` for defining the numeric value of that option.